data/hai/hai reveal 1 intro.txt

```diff
@@ -58,11 +58,49 @@ mission "Event: Pug Media Coverage"
+			`As you leave your ship, you see two captains ranting about a video playing on a datapad. The walkway forces you to pass close to them.`
+			choice
+				`	"Hey! What's this video about?"`
+				`	(Leave them be.)`
+					decline
+			`	You're startled when the captains turn to you, both with preposterous sideburns and rat-like mustaches. It's impossible to tell if they're smiling or frowning, as the pair live squarely in the middle of the uncanny valley. They appear to be twins, as are their well-armed Headhunters.`
+			`	The left twin asks, "Can you believe this? It's been weeks and everyone's still obsessing over the Pug invasion. The news was all experts and officials at first, but now we just see talking heads and conspiracy theorists."`
+			`	In disbelief, the right twin responds, "It's just a propaganda exercise to boost Syndicate sales. Even if these elusive 'Pug' existed, we won't hear the end of it as long as the story boosts public opinion of the Syndicate."`
+			choice
+				`	"The past few months have brought some of the biggest changes in recent history. Of course people are going to keep talking about it."`
+				`	"Don't you see this is what we need? The Pug invasion was a ploy to cause humanity to band together and learn to resolve differences without conflict."`
+				`	"Oh, the Pug were real alright. I met some and fought them."`
+					goto nobody
+				`	"I think you both need to shave."`
+					goto scowl
+			`	The right twin rolls his eyes while the left twitches his mustache awkwardly and responds. "Look at this bozo! Bounty's Councilor Maduena Kabiru keeps ranting about defending against unknown dangers at the fringes of human space. Non-stop! Don't we have enough immediate problems without imagining new horrors?"`
+			`	"Like terrorists nuking us," says the right twin.`
+			choice
+				`	"No, don't worry about that. I ended the bomb threat."`
+				`	"There's a lot of dangers out there, but don't worry. I'll be there to fight them off."`
+				`	"Maybe you're right."`
+					goto shrug
+				`	"All we can do is get on with our lives, right?"`
+					goto shrug
+				`	"If you don't shave soon, people will think you're wearing masks."`
+					goto scowl
+			label nobody
+			`	The right twin says, "Who are you, Admiral Danforth?" The left one laughs and adds, "Did the Navy fire you or are you here on holiday?" Both of them laugh for a few seconds before you can respond.`
+			choice
+				`	"No. I'm the famous Captain <first> <last>!"`
+				`	"You don't recognize me?"`
+					goto shrug
+				`	"Never mind."`
+					goto shrug
+				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
+					goto scowl
+			`	Both twins look confused and say in unison, "Who?" They shrug and walk away, leaving you to wonder whether your doings are hidden in a cloud of government-enforced secrecy, or if some people have already forgotten about you.`
```

Comment:
The "you are not the center of the universe" thing just means that if you do another questline that's mutually exclusive or locks you out of something while still progressing to a later stage that events which would happen either way still happen, it means that the people responsible for things happening are the same, whether or not you are involved in finding them, it means that you cannot change the reality of a place before you get there by choosing one path or another through a narrative. It's about consistency within the world regardless of how involved or not you are, as you progress through comparable narratives through events.

It doesn't simply mean that you can't be famous or recognised as important.

As an example context for this, knowing about the Hai before hand does not, and will not, provide you an option to somehow know which random crewman is going to go to Tarazed and kick this off. It kicks off regardless of you.
Likewise, it *might* come about that there's a whole other pathway through this in the future if you've done Unfettered plot beforehand, but it won't change that internal pressure is likely to see the Hai make an issue about the kidnappings and result in Stormhold getting raided - after all Teeneep has her concerns that Sayari knows about before you ever get involved.
Some things you can affect, and some things you're along for the ride with because they're driven by outside forces.

Realistically your name is in the news from this. You are "famous".

*However,* that doesn't mean that they actually care about you. These guys sound like they're from paradise worlds judging by their reference to getting nuked as a terrorist threat. They've drunk the koolaid. You're an ex-FW councillor with a military involvement in an event that one of them believes is propaganda. 
They've probably *heard* of you - but that doesn't mean that have the least bit respect for you, or wouldn't gladly attribute any of your heroics to Danforth as their preferred celebrity figure.

Reply:
Being a war hero does not make you a household name. In war, it is those who lead the people that are most visible. We remember Churchill and Hitler, but most people won't know the name of another consequential captain, Paul Tibbets. Captain Last isn't discussing things daily with Parliament or the Senate, or being interviewed by media figures. Political buffs may recognize "Captain First Last" for a long time, but that name will quickly vanish from the public consciousness. 

In the updated story, the hairy twins explain this to you.